src/wt/main.js

```diff
@@ -1,5 +1,51 @@
+import { Worker } from 'worker_threads';
+import { cpus } from 'os';
+import { dirname } from 'path';
+import {fileURLToPath} from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+
+const __dirname = dirname(__filename);
+
 const performCalculations = async () => {
-    // Write your code here
+
+    const arResult = [];
+
+    const workerResult = (data) => {
```

Comment:
This function is redundant I think. I will work you simply `Promise.allSettled(promises).then(list => console.log(list)`;